CHANGELOG.md

```diff
@@ -2,6 +2,86 @@
+        GlyphsRasterizationOptions.Builder()
+          .rasterizationMode(GlyphsRasterizationMode.IDEOGRAPHS_RASTERIZED_LOCALLY)
+          .build()
+      )
+      .build()
+
+    // set token and cache size for this particular map view
+    val resourceOptions = ResourceOptions.Builder()
+      .accessToken(getString(R.string.mapbox_access_token))
+      .cacheSize(75_000L)
+      .build()
+
+    // set initial camera position
+    val initialCameraOptions = CameraOptions.Builder()
+      .center(Point.fromLngLat(-122.4194, 37.7749))
+      .zoom(9.0)
+      .build()
+
+    val mapboxMapOptions =
+      MapInitOptions(this, resourceOptions, mapOptions, initialCameraOptions, true)
+    // create view programmatically
+    customMapView = MapView(this, mapboxMapOptions)
+  ```
+## Features ✨ and improvements 🏁
+* [animation] High-level animations return cancelable interface ([#262](https://github.com/mapbox/mapbox-maps-android/pull/262))
+* Introduce OfflineManager API that manages style packs and produces tileset descriptors for the tile store.
+
+The new API replaces the deprecated OfflineRegionManager API. The OfflineManager API can be used to create offline style packs that contain style data, such as: style definition, sprites, fonts and other resources. Tileset descriptors created by the OfflineManager API are used to create tile packs via TileStore API. Mobile maps SDKs use tile packs for rendering map content. Example of how to use API is presented in a following snippet (pseudocode):
+ ```
+  // 1. Get an instance of OfflineManager.
+  val offlineManager = OfflineManager(map.getResourceOptions())
+
+  // 2. Create a tile region with tiles for Streets
+  TilesetDescriptorOptions tilesetOptions(
+      "mapbox://styles/mapbox/streets-v11",
+      0, // min zoom
+      5, // max zoom
+  );
+
+  // Creation of a tileset descriptor WILL implicitly create a style package for provided StylePackLoadOptions.
+  tilesetOptions.stylePackOptions = StylePackLoadOptions(GlyphsRasterizationMode.AllGlyphsRasterizedLocally);
+  val descriptor = offlineManager.createTilesetDescriptor(tilesetOptions);
+
+  val tileStore = TileStore.getInstance();
+  TileRegionLoadOptions tileRegionLoadOptions(geometry, [descriptor]);
+  tileStore.loadTileRegion("my_region", tileRegionLoadOptions,
+    onProgressCallback(result) {
+      // Handle TileRegionLoadProgress result
+    },
+    onFinishedCallback(result) {
+      if (result) {
+        // Handle TileRegion result
+      } else {
+        // Handle TileRegionError
+      }
+    }
+  );
+  ```
+* Tile Store
+  - Add index for group <-> tiles to speed up eviction queries
+  - Don't return cached responses for 401/403/Unauthorized requests
+  - The invoked async methods complete even if the client code does not keep the tile store instance any more.
+## Bug fixes 🐞
+* Keep CompassPlugin enabled/disabled state after other properties update ([#252](https://github.com/mapbox/mapbox-maps-android/pull/252))
```

Comment:
should also include  https://github.com/mapbox/mapbox-maps-android/pull/273

Reply:
should also include https://github.com/mapbox/mapbox-maps-android/pull/279

Reply:
and https://github.com/mapbox/mapbox-maps-android/pull/228